packages/forms/src/widgets/EnumerationWidget/utils/utils.js

```diff
@@ -0,0 +1,60 @@
+// private
+function selectAllBetween(min, max, items) {
+	return items
+		.filter((item, index) => index >= min && index <= max)
+		.map((item, index) => index);
+}
+
+export function manageCtrlKey(item, selectedItems) {
+	const itemFound = selectedItems.find(currentItem => currentItem === item.index);
+	// if index present, remove it
+	if (itemFound !== undefined) {
+		return selectedItems.filter((indexSelected) => indexSelected !== item.index);
+	}
+
+	selectedItems.push(item.index);
+
+	return selectedItems;
+}
+
+export function manageShiftKey(item, selectedItems, items) {
+	let result = selectedItems;
```

Comment:
You can remove this line or set selectedItems in a variable more talking. We lost with this name what result contains.